RetroAchievements/RAWeb
# feat(analytics): track gamelist type and forum topic id

Fixes a bug where the gamelist type is not being tracked properly. Also sends along what forum topic ID users are viewing.

**How to test this PR:**
Open your browser's network tab. View your Want to Play Games List. You should see the `type` prop being sent along in the pageview event:
![Screenshot 2024-08-28 at 8 54 01 AM](https://github.com/user-attachments/assets/a89b1bab-27f9-4141-a705-6e87403aef94)

Open any forum topic. You should see the `topicId` prop being sent along in the pageview event:
![Screenshot 2024-08-28 at 8 54 33 AM](https://github.com/user-attachments/assets/3dd0ebc3-336f-4042-8f99-2dba294b808d)

It's okay that IDs are passed along as strings.
